### ISSUE TITLE
Having Jounin and ANBU being shiken related rather than appointed by Kage

### ISSUE THREAD

**Issue by kauedavila** [IllusiveBIair/Naruto-Evolution-Community#58]
Being able to take A Ranks should be considered an end game kind of content, rather than being able to take it as soon as you get Chunnin. Making it so you would have to grind your way up to Jounin / Anbu instead of having kages give it away might be a good change to the game progression.

**Comment by RTDjay** [comment]
I agree, I think kages should still appoint it though but are only able to when the chuunin are higher levels such as 45-50+ for jounin and 60+ for anbu

**Comment by IllusiveBIair** [comment]
The trouble with this is that (especially after the prestige system is out) people will be at varying degrees of power at different times. So for example when the wipe happens from their content being locked behind level would mean the early days of the wipe would be significantly less enjoyable. The idea is that if a Kage is hiring weak jounin and anbu, the missions that those people do will fail and they will feed the enemy. But currently people aren't even trying to scout for other peoples escorts or intel missions. I plan to make it so that they are announced to the world so that people will be more inclined to contest them. This should make the requirement for being a jounin or anbu much higher. It will be down to the kage to be appointing the right people to the right role. What are your opinions on this?

**Comment by kauedavila** [comment]
To be honest it pretty much depends on the playerbase on the first days. As soon as we have Kages, i doubt any of them would choose to not give it to anyone who would ask for it. People dont really play along with their villages also, i personally would try and change village to the one willing to give me Jounin asap. The way the combat plays along, the number of players in a village matters more than if a Kage tries to avoid feeding the other. Anyway we dont know much about the prestige system yet, so it could all be small problems close to how good prestige might be. 
   Independent on how Jounin plays out, atleast ANBU could be not given by Kages, but conquered through your own efforts. Something like a village reputation system or simply leveling. The less we depend on other players to access the contents of the game the better.

**Comment by IllusiveBIair** [comment]
It's definitely something to consider. As for the number of players online in each village, I recently implemented a system where you get more exp from missions the more outnumbered you are in terms of players online. Since implementing that system I have seen the player numbers online be far more balanced between the two villages than ever before. This should go a long way to pushing things towards more where we want them to be. As for a reputation system, in your opinion, what exactly would you envision that looking like?
